app/javascript/stylesheets/editor.scss

```diff
@@ -2,7 +2,7 @@
   section {
     border: 1px solid gray;
     border-radius: .25rem;
-    background-color: $floral-white;
+    background-color: $stanford-fog-light;
```

Comment:
A change I introduced to further align the app with the brand identity. Colors are very close.